### DISCUSSION TITLE
Size and style to be as values

### DISCUSSION THREAD

**Discussion by vlados** [wireui/wireui#106] [Ideas]
I think it will be better for readability of the code, if the style and size to be as html attributes with values
Before:
```php
<x-button primiray md />
```
After:
```php
<x-button style="primary" size="md" />
```

**Comment by ph7jack** [comment]
I didn't understand what you meant

**Reply by vlados** [reply]
Updated the question

**Reply by ph7jack** [reply]
https://livewire-wireui.com/docs/buttons#button-options
Look the button options, is possible to pass as your example
